templates/leader_board.html

```diff
@@ -50,7 +50,7 @@ <h1 style="font-size: 60pt; font-family: 'Staatliches', cursive; color: rgb(22,
     <div class="card">
         <h5 class="card-header"
             style="font-size: 30pt; font-family: 'Staatliches', cursive; color: white; background-color: black;">
-            {{i + 1}}. {{users[i].user_name}}</h5>
+            #{{i + 1}}&ensp; {{users[i].user_name}}</h5>
```

Comment:
This is a good mod for the leaderboard adding the number and pound sign next to each user